guides/source/documents.yaml

```diff
@@ -132,11 +187,18 @@
     -
       name: The Rails Command Line
       url: command_line.html
-      description: This guide covers the command line tools provided by Rails.
+      description: >
+        There are a few commands that are absolutely critical to your everyday
+        usage of Rails. This guide covers the command line tools provided by
+        Rails.
     -
       name: The Asset Pipeline
       url: asset_pipeline.html
-      description: This guide documents the asset pipeline.
+      description: >
+        The asset pipeline provides a framework to concatenate and minify or
+        compress JavaScript and CSS assets. It also adds the ability to write
+        these assets in other languages and pre-processors such as
+        CoffeeScript, Sass, and ERB.
```

Comment:
Perhaps this description should also mention images, which are a big part of the asset pipeline as well?

Reply:
Done!